### DISCUSSION TITLE
Nice job on the anti-phishing feature!

### DISCUSSION THREAD

**Discussion by WhamboMPS** [simple-login/app#991] [Kind words ❤️]
Just received my first message forwarded via SimpleLogin that included the new anti-phishing warning message.  Wasn't previously aware that this feature had been released and was really appreciated that it existed. Also appreciated the hyperlink to the [help document on this feature](https://simplelogin.io/docs/getting-started/anti-phishing/). Keep up the good work, @[nguyenkims](https://github.com/nguyenkims).

![SL Anti-Phishing Message](https://user-images.githubusercontent.com/88680678/168481634-a4e6c131-2ed2-408c-912c-d3cdd4a78ff5.jpg)

**Comment by nguyenkims** [comment]
Thanks @WhamboMPS and glad that you like it! We found a way to add a message to the email body without breaking its format :).
